packages/web-components/src/switch/switch.styles.ts

```diff
@@ -0,0 +1,175 @@
+import { css } from '@microsoft/fast-element';
+import {
+  colorCompoundBrandBackground,
+  colorCompoundBrandBackgroundHover,
+  colorCompoundBrandBackgroundPressed,
+  colorNeutralBackgroundDisabled,
+  colorNeutralForeground1,
+  colorNeutralForeground3,
+  colorNeutralForeground3Hover,
+  colorNeutralForeground3Pressed,
+  colorNeutralForegroundDisabled,
+  colorNeutralForegroundInverted,
+  colorNeutralForegroundInvertedHover,
+  colorNeutralForegroundInvertedPressed,
+  colorNeutralStrokeAccessible,
+  colorNeutralStrokeAccessibleHover,
+  colorNeutralStrokeAccessiblePressed,
+  colorNeutralStrokeDisabled,
+  colorTransparentBackground,
+  curveEasyEase,
+  durationNormal,
+  fontFamilyBase,
+  fontSizeBase300,
+  fontWeightSemibold,
+  lineHeightBase300,
+  spacingHorizontalM,
+  spacingHorizontalMNudge,
+  spacingHorizontalS,
+  spacingHorizontalXXS,
+  spacingVerticalMNudge,
+  spacingVerticalS,
+  spacingVerticalXS,
+} from '../theme/design-tokens.js';
+
+export const styles = css`
+  :host([hidden]) {
+    display: none;
+  }
+  :host {
+    display: inline-flex;
+    align-items: center;
+    outline: none;
+    user-select: none;
+  }
+  :host([labelposition='before']) {
+    flex-direction: row-reverse;
+  }
+  :host([labelposition='above']) {
+    flex-direction: column-reverse;
+    align-items: flex-start;
+  }
+  :host([disabled]) .label,
+  :host([readonly]) .label,
+  :host([readonly]) .switch,
+  :host([disabled]) .switch {
+    cursor: not-allowed;
+  }
+  :host .status-message {
+    color: ${colorNeutralForeground1};
+    line-height: ${lineHeightBase300};
+    font-size: ${fontSizeBase300};
+    font-weight: ${fontWeightSemibold};
+    font-family: ${fontFamilyBase};
+    padding: ${spacingVerticalXS} ${spacingHorizontalS} ${spacingVerticalXS} ${spacingHorizontalS};
+  }
+  :host([disabled]) .status-message {
```

Comment:
@chrisdholt Did I see that we removed the on/off status message?

Reply:
It's in PR here: https://github.com/microsoft/fast/pull/6609